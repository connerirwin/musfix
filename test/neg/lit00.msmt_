; Sorts
(declare-sort Peano 0)

; Constants
(declare-const _One Peano)
(declare-const _Zero Peano)

(assert (distinct _One _Zero))

; Uninterpreted Functions


; Qualifiers


; Well-formedness Constraints


;  Constraints
(constraint (forall ((xOne Peano)(xZero Peano)) (=> (and (= xZero _Zero) (= xOne _One)) (= xZero xOne))))

