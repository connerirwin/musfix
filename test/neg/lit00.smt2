; Uninterpreted Sorts
(declare-sort List 1)
(declare-sort Peano 0) ; why wasn't this emitted

; Constants
(declare-const one Peano)
(declare-const zero Peano)


; Distinct Constants
(assert (distinct one zero))


; Uninterpreted Functions


; Qualifiers


; Well-formedness constraints


; Horn constraints
(constraint
  (forall ((xZero Peano) (xOne Peano) (v1 Int))
    (=> (and (= xZero zero) (= xOne one))
        (= xZero xOne))))
