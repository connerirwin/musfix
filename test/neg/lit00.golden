
Preparing to find greatest fixpoint...

Inputs
--------
Reading from file test/neg/lit00.smt2
SortDecl "List" 1
SortDecl "Peano" 0
ConstantDecl "one" (DataS "Peano" [])
ConstantDecl "zero" (DataS "Peano" [])
DistinctDecl ["one","zero"]
HornConstraint [Var (DataS "Peano" []) "xZero",Var (DataS "Peano" []) "xOne",Var IntS "v1"] (Binary Implies (Binary And (Binary Eq (Var (DataS "Peano" []) "xZero") (Func (DataS "Peano" []) "zero" [])) (Binary Eq (Var (DataS "Peano" []) "xOne") (Func (DataS "Peano" []) "one" []))) (Binary Eq (Var (DataS "Peano" []) "xZero") (Var (DataS "Peano" []) "xOne")))

QMAP
--------
fromList []

Candidates
--------
[]


Final candidates: 

